### DISCUSSION TITLE
Feature: flags of server region on server / player list.

### DISCUSSION THREAD

**Discussion by wolfreak99** [OpenRCT2/OpenRCT2#15676] [Feature requests]
I think this would be a nice feature to help localize players to more proper servers. It could be of the origin they're located at, or it could be just based on the language the server is set to.

**Comment by Ziscor** [comment]
Bump.

I think this would be very helpful, as people prefer to join servers that are geographically close to them, as that'd improve ping times and latency with the server. 

Maybe something like this?:
![tokusentai](https://cloud.githubusercontent.com/assets/18721353/21961126/6bc699b2-db27-11e6-81b8-81d27030dc8a.png)

**Comment by wolfreak99** [comment]
where did you find those icons? those are so small and cute and would work perfectly!

**Comment by AaronVanGeffen** [comment]
They look like the [famfamfam flag icons](http://www.famfamfam.com/lab/icons/flags/) to me.
